arch/ecmwf/aurora/nec/4.0.0/toolchain.cmake

```diff
@@ -0,0 +1,89 @@
+
+####################################################################
+# COMPILER
+####################################################################
+
+include( /opt/nec/ve/share/cmake/toolchainVE.cmake )
+
+
+set( ECBUILD_FIND_MPI ON )
+
+####################################################################
+# Enviroment Variables
+####################################################################
+set(NMPI_ROOT /opt/nec/ve/mpi/2.23.0)
+
+####################################################################
+# OpenMP FLAGS
+####################################################################
+
+set( OpenMP_C_FLAGS             "-fopenmp " )
+set( OpenMP_CXX_FLAGS           "-fopenmp " )
+set( OpenMP_Fortran_FLAGS       "-fopenmp " )
+
+####################################################################
+# OpenAcc FLAGS
+####################################################################
+
+set( OpenACC_Fortran_FLAGS "-acc -ta=tesla:lineinfo,deepcopy,maxregcount:100,fastmath" )
```

Comment:
These look suspiciously like Nvidia GPU flags; does NEC really support OpenACC, or are these development/debug leftovers?

Reply:
Sorry, they are leftovers that I forgot to remove them.